lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -193,3 +193,20 @@ class _OverReactReduxDevToolsMiddleware extends MiddlewareClass {
 /// );
 /// ```
 final MiddlewareClass overReactReduxDevToolsMiddleware = _OverReactReduxDevToolsMiddleware();
+
+/// A Middleware that can be added to a [DevToolsStore] in order to utilize the Redux DevTools browser extension.
+/// Similar to [overReactReduxDevToolsMiddleware], but allows passing of options to initialize dev tools with.
+///
```

Comment:
Could we document what `name` does?

```dart
/// Arguments:
/// - [name] - the instance name to be shown on the monitor page. Default value is `document.title`. 
///   If not specified and there's no document title, it will consist of tabId and instanceId. 
///   Use this if your page has multiple stores.
```